cdc/scheduler/internal/tp/agent.go

```diff
@@ -35,25 +35,47 @@ import (
 var _ internal.Agent = (*agent)(nil)
 
 type agent struct {
+	agentInfo
 	trans transport
 
 	tableM *tableManager
 
-	// owner's information
 	ownerInfo ownerInfo
+}
 
-	// maintain the capture's information
+type agentInfo struct {
 	version      string
 	captureID    model.CaptureID
 	changeFeedID model.ChangeFeedID
 	epoch        schedulepb.ProcessorEpoch
 
 	// Liveness of the capture.
 	//
-	// CaptureLivenessShuttingDown rejects all add table requests.
+	// LivenessCaptureStopping rejects all add table requests.
 	liveness model.Liveness
 }
 
+func (a agentInfo) String() string {
+	bytes, _ := json.Marshal(a)
+	return string(bytes)
```

Comment:
What does it look like? I think json skips unexported fileds, so it's blank in this case.